### PR TITLE
added initial readme with a bit more content

### PR DESCRIPTION
**Pull request by jackdotwa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +8,129 @@
 A dataframe auditor that computes a number characteristics of the data. This is done by profiling the frame.
```
Comment (berndtlindner) [inline README.md]: "Profiling the frame". Rather say "dataframe"

File: README.md [hunk 2]
```diff
@@ -7,3 +8,129 @@
...
+>
+> [Testing](#testing)
+>
+> [Usage](#usage)
+> 
+> [Contributions](#contributions)
+
+## Summary
+
+  Data profiling is important in data analysis and analytics, as well as in determining characteristics of data pipelines.
```
Comment (berndtlindner) [inline README.md]: [Data profiling](https://en.wikipedia.org/wiki/Data_profiling)

File: README.md [hunk 3]
```diff
@@ -7,3 +8,129 @@
...
+  This repository aims to provide a means to extract a selection of attributes from data.
+  
+  It is currently focused on processing _pandas_ dataframes, but this functionality is being
```
Comment (berndtlindner) [inline README.md]: Specifically say python?

Comment (jackdotwa) [reply README.md]: It is clear from repo tags/display in github I feel. Also, pandas, so not going to fuss about it.

File: README.md (hunk 1 above)
Comment (jackdotwa) [reply README.md]: removed `A dataframe auditor that computes a number characteristics of the data. This is done by profiling the frame. `

File: README.md (hunk 2 above)
Comment (berndtlindner) [reply README.md]: sorry I meant you should put url/link like i did perhaps

File: README.md (hunk 3 above)
Comment (berndtlindner) [reply README.md]: cool

File: README.md (hunk 2 above)
Comment (jackdotwa) [reply README.md]: ah. cool cool cool. no doubt no doubt

Comment (jackdotwa) [reply README.md]: Added link. interestingly, we are using it in more continual-validation style approach, but yeah, that link does speak to the roots.

**Pull request merged**